src/content/app-platform/namespace-configuration/index.md

```diff
@@ -0,0 +1,76 @@
+---
+linkTitle: App target namespace metadata
+title: How to annotate an app's target namespace via its App CR
+description: Configuring metadata for the target namespace of an app via its app CR so it can be used by other tools such as service meshes.
+weight: 20
```

Comment:
```suggestion
weight: 40
```

This controls the order on https://docs.giantswarm.io/app-platform/ 

This feature is great but only applies to some app CRs. Can you also bump `apps` to 50?

Reply:
Thanks, TIL this is ordering by descending order.